includes/User.php

```diff
@@ -2585,7 +2650,41 @@ public function getGlobalFlag($flag, $default = null) {
 	 * @return bool
 	 */
 	public function setGlobalFlag($flag, $value) {
-		$this->setOption($flag, $value);
+		$this->setOptionHelper($flag, $value);
+	}
+
+	/**
+	 * Create a local option name. All localized (wikia specific) options,
+	 * preferences, attributes or flags should be of the form "{option}-{cityId}"
+	 *
+	 * IF YOU USE $sep, MAKE A PLAN TO NORMALIZE IT TO "-"!
+	 *
+	 * @param string $property
+	 * @param int $cityId [optional]
+	 * @param string $sep the separator between the option and the id.
+	 * @return string
+	 */
+	public static function localToGlobalPropertyName($property, $cityId = null, $sep = '-') {
```

Comment:
Hm, yeah, I guess it can be seen that way. The way I was looking at it was "Here's a local property for this wiki, how can I reference this globally?"